src/components/Modal/Modal.jsx

```diff
@@ -58,7 +77,7 @@ class Modal extends React.Component {
     }
   }
 
-  handleDocumentKeyDown = event => {
+  handleDocumentKeyDown(event) {
```

Comment:
:thinking: So these are different statements.

The former arrow function binds `this` to the current `this` context (creation phase). The "normal" class method syntax `handleDocumentKeyDown(event)` doesn't bind `this` to anything, so it'll change based on whatever calls it.

In general, I've found it less confusing to stick with the arrow functions so `this` doesn't change.

What was the reason for changing from an arrow function to a "normal" function?

Reply:
```js
handleDocumentKeyDown = event => {
```
...gives me this eslint error:

```
Parsing error: Unexpected token = [79, 25]
```

Reply:
@cvalarida -

This error (predictably) goes away when I add this to my eslintconfig:

`"parser": "babel-eslint",`

...and I do get the error when I pop open another component using an arrow function (like DropDownPanel).  

![image](https://user-images.githubusercontent.com/62304138/108437772-f54bc880-721b-11eb-82b5-cfd1d761e6d0.png)

Reply:
Ahh, okay. So this is a linting issue then. I'll make a PR for that today (hopefully). These declarations probably shouldn't be changed, though.

Reply:
Sounds good to me @cvalarida.  Let me know when I should update my branches.